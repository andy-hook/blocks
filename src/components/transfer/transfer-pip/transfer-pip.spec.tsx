import React from "react"
import TransferPip from "./transfer-pip"
import { render } from "@test-utils"

jest.mock("react-truncate-string")

describe("<TransferPip />", () => {
  it("renders correctly", () => {
    const tree = render(<TransferPip />)
    expect(tree).toBeTruthy()
  })
})
