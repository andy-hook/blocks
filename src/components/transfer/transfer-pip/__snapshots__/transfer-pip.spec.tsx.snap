// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TransferPip /> renders correctly 1`] = `
<div
  className="sc-htpNat frRoNZ"
>
  <div
    className="sc-bdVaJa kasXVf"
    size="sm"
  />
</div>
`;
